src/lib.rs

```diff
@@ -70,13 +85,37 @@ impl<E: JubjubEngine> FullViewingKey<E> {
             ovk: xsk.ovk,
         }
     }
+
+    fn to_bytes(&self) -> [u8; 96] {
+        let mut result = [0u8; 96];
```

Comment:
I couldn't find what `0u8` means in rust :)

Reply:
It is just explicitly specifying that the type of `0` is `u8`. It's not usually necessary, as the compiler in most cases can elide the type from context, but it doesn't hurt.